cluster/gce/gci/master-helper.sh

```diff
@@ -62,7 +62,17 @@ function replicate-master-instance() {
   kube_env="$(echo "${kube_env}" | grep -v "ETCD_PEER_CERT")"
   kube_env="$(echo -e "${kube_env}\nETCD_PEER_CERT: '${ETCD_PEER_CERT_BASE64}'")"
 
-  echo "${kube_env}" > ${KUBE_TEMP}/master-kube-env.yaml
+  kube_env="$(echo "${kube_env}" | grep -v "_HASH")"
+
+  # Add default kube-env parameters when they are not presented in ${kube_env}
+  # It might happen when we adding master with different version (skewed cluster).
+  build-kube-env true "${KUBE_TEMP}/master-kube-env-default.yaml"
+  echo "${kube_env}" > "${KUBE_TEMP}/master-kube-env-existing.yaml"
+  awk -F: 'NR==FNR{a[$1];print;next} !($1 in a)'\
```

Comment:
Do I understand correctly that this is merging those two files by taking all things from both files but if some exists in both taking it from "existing"?
Can you please add a comment for people who don't know awk well like me to understand what is happening here?